src/main/java/uk/gov/hmcts/reform/em/hrs/service/ccd/CcdDataStoreApiClient.java

```diff
@@ -77,7 +83,7 @@ public Long createCase(final UUID recordingId, final HearingRecordingDto hearing
                     "Create New Case"
                 );
             }
-
+            LOGGER.error("Error Creating Case", e);
```

Comment:
As seen on the Sonar report for this PR build, this adds a code smell. Could you please remove this log error line?

Reply:
will remove